lab-max/app/component/gallery/gallery-list/gallery-list.html

```diff
@@ -0,0 +1,16 @@
+<li class="gallery-li">
+  <h3> {{galleryListCtrl.gallery.name}} </h3>
+  <p> {{galleryListCtrl.gallery.desc}} </p>
```

Comment:
@meeksfred general comment - I've noticed that much of your code (and Duncan's to be fair) includes spaces before and after Angular expressions and general text content when nested inside of an element.  HTML ignores whitespace, except for 1 single space, which can cause styling issues.  If you need a space between text, CSS can (and should) control that.